designer/dgraph.js

```diff
@@ -241,6 +241,7 @@ async function db(contextData) {
 }
 
 process.on('unhandledRejection', (reason, promise) => {
+   queryForEntity();
   console.log('Unhandled Rejection at:', promise, 'reason:', reason);
```

Comment:
[major]This log need to be above  queryForEntity() function with proper message like retrying.

Reply:
done